hare/algorithm.go

```diff
@@ -281,9 +286,16 @@ PreRound:
 		case <-ticker.C: // next round event
 			proc.onRoundEnd()
 			proc.advanceToNextRound()
+
+			// exit if we reached the limit on number of iterations
+			if proc.k/4 > int32(proc.cfg.LimitIterations) {
```

Comment:
why not => ?
First iteration is 0, right?

Reply:
first iteration is 0 so if I say no more than x=1 iterations then when we move to second iteration we will stop...